1.0/Dockerfile

```diff
@@ -38,5 +28,9 @@ RUN /tmp/build.sh
 
 ENV LD_PRELOAD /opt/td-agent/embedded/lib/libjemalloc.so
 
+RUN /opt/fusion/bin/fusion start
+
+RUN service td-agent restart
```

Comment:
Wouldn't read the updated config until restarted. Perhaps a difference in the newer version of td-agent?